MREUnityRuntime/MREUnityRuntimeLib/Core/PhysicsBridge.cs

```diff
@@ -0,0 +1,229 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MixedRealityExtension.Core
+{
+	public class PhysicsBridge
```

Comment:
I think some code comments would be usefull

Reply:
on the functions level and maybe not for all functions

Reply:
Agree, that's a big todo.